elyra/metadata/schemas/component-registry.json

```diff
@@ -1,9 +1,8 @@
 {
   "$schema": "https://raw.githubusercontent.com/elyra-ai/elyra/master/elyra/metadata/schemas/meta-schema.json",
   "$id": "https://raw.githubusercontent.com/elyra-ai/elyra/master/elyra/metadata/schemas/component-registry.json",
-  "title": "Component Registry",
+  "title": "Pipeline Component",
```

Comment:
Ah good point. Yeah, I think that would be the best way to handle it!

Reply:
Actually, I guess since this particular schema will be deprecated as of the BYO-catalog PR, it would be better to adjust the frontend title to `Component Catalogs`, which I think is what it is now